sky/clouds/service_catalog/aws_catalog.py

```diff
@@ -11,8 +11,12 @@
 if typing.TYPE_CHECKING:
     from sky.clouds import cloud
 
-_df = common.read_catalog('aws/vms.csv')
-_image_df = common.read_catalog('aws/images.csv')
+_UPDATE_FREQUENCY_HOURS = 7
```

Comment:
nit: add a comment, and a note on keeping this the same as skypilot-catalog's update frequency?

nit: (minor preference) consider replacing 'update' with 'pull'.